doc/conf.py

```diff
@@ -36,7 +36,8 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx_copybutton'
+    'sphinx_copybutton',
+    'sphinx_dunedomain',
```

Comment:
do we need to update any dependencies somewhere for this? E.g. in nix.

Reply:
I managed to build the doc with `nix develop .#doc -c make doc` so I guess no.

Reply:
No, this refers to `doc/exts/sphinx_dunedomain.py` which only uses sphinx, so there are no new deps. (and I'm using nix to develop this)